distros/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM alpine:3.5
```

Comment:
There is no `distros/Dockerfile` file, please move this into the top directory replacing the old one.